concordium-std/src/test_infrastructure.rs

```diff
@@ -971,7 +934,8 @@ mod test {
             false
         );
 
-        let mut iter = state.get::<_, StateSet<u8, _>>(my_set_key).unwrap().unwrap().iter();
+        let mut iter =
+            state.get::<_, StateSet<u8, _>>(my_set_key).unwrap().unwrap().iter().unwrap();
```

Comment:
Please use `expect` and explain why unwrapping is intended. Otherwise the error message will be hard to understand.